plugins/hls-alternate-number-format-plugin/src/Ide/Plugin/AlternateNumberFormat.hs

```diff
@@ -0,0 +1,134 @@
+{-# LANGUAGE DataKinds                  #-}
+{-# LANGUAGE DeriveGeneric              #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE TypeFamilies               #-}
+{-# LANGUAGE TypeOperators              #-}
+
+module Ide.Plugin.AlternateNumberFormat where
+
+import           Control.Lens                         ((^.))
+import           Control.Monad.Except                 (ExceptT, MonadIO, liftIO)
+import qualified Data.HashMap.Strict                  as HashMap
+import           Data.Maybe                           (fromMaybe, isJust)
+import           Data.Text                            (Text)
+import qualified Data.Text                            as T
+import           Debug.Trace                          (traceM)
```

Comment:
hopefully not used anywhere?

Reply:
Sigh, I had to turn off -Wall for a period and I forgot to reenable it.

Reply:
Cleaned up a bunch of imports...